### ISSUE TITLE
Android: keyboard for date fields

### ISSUE THREAD

**Issue by BaumiCoder** [CollaboraOnline/online#5322]
**Is your feature request related to a problem? Please describe.**
I have an table in an spreadsheet with a date column. I want to type in dates in there and most of the time it is the date of today. I have make the fields in this column formated as date, but I can type in any stuff (also letters).

**Describe the solution you'd like**
I want to have an input method or keyboard for date fields in spreadsheets. This should have numbers and dateseperator ("." in most countries) to type the date and some smart keys like "today", "tomorrow" and "yesterday".

**Describe alternatives you've considered**
Maybe there would be also a way to tell the android system, that it is date field and the android system provide an date picker.

**Comment by Hoggy-1** [comment]
I agree that this is very useful. I have used Apple's Numbers and the available, adjusting keyboard layouts improve massively the usability on mobile devices. I am quite new to this topic of app specific keyboard layouts and I don't know whether this is the responsibility of the app or the system to provide such layouts.
